components/Templates/Templates.php

```diff
@@ -395,6 +395,23 @@ public function add_meta_boxes() {
 					'restrict_capability' => true,
 				),
 			),
+			array(
+				'name'       => 'show_restrict_message',
+				'label'      => __( 'Show no access message?', 'pods' ),
+				'default'    => 1,
+				'type'       => 'boolean',
+				'dependency' => true,
+			),
+			array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 556194f118e48b5841b50130997da2d3</sub>